ocfweb/docs/docs/staff/procedures/vhost.md

```diff
@@ -59,6 +60,7 @@ Then, whether or not the group has web virtual hosting, request the following
 DNS records:
 
     hostname.Berkeley.EDU. IN A 169.229.226.23
+    hostname.Berkeley.EDU. IN AAAA 2607:f140:8801::1:23
     hostname.Berkeley.EDU. IN MX 5 anthrax.OCF.Berkeley.EDU.
```

Comment:
maybe add a line about "they might already have it" since the current instructions always set it up

this might've prevented accidentally emailing hostmaster last week